config.json

```diff
@@ -0,0 +1,232 @@
+﻿{
+  "settings": {
+    "SkipTests":{
+      "description": "Enables/Disables running tests.",
+      "valueType": "property",
+      "values": ["True", "False"],
+      "defaultValue": false
+    },
+    "OSGroup": {
+      "description": "OS Group for result binaries.",
+      "valueType": "property",
+      "values": ["AnyOS", "Windows_NT", "Linux", "OSX", "FreeBSD", "NetBSD"],
+      "defaultValue": "AnyOS"
+     },
+    "RestoreDuringBuild": {
+      "description": "Enables/Disables the restoration of packages.",
+      "valueType": "property",
+      "values": ["True", "False"],
+      "defaultValue": true
+    },
+    "Platform": {
+      "description": "Enables/Disables the restoration of packages.",
+      "valueType": "property",
+      "values": ["AnyCPU", "x86", "arm", "x64", "amd64"],
+      "defaultValue": "x64"
+    },
+    "ConfigurationGroup": {
+      "description": "Enables/Disables the restoration of packages.",
+      "valueType": "property",
+      "values": ["Release", "Debug"],
+      "defaultValue": "Debug"
+    },
+    "BuildPackages": {
+      "description": "Enables/Disables building packages.",
+      "valueType": "property",
+      "values": ["True", "False"],
+      "defaultValue": true
+    },
+    "BuildTests": {
+      "description": "Enables/Disables building tests.",
+      "valueType": "property",
+      "values": ["True", "False"],
+      "defaultValue": true
+    },
+    "MsBuildLogging": {
+      "description": "MsBuild logging options.",
+      "valueType": "passThrough",
```

Comment:
That the value that you have is not going to be processed or modified. It would print it as it is.